src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -85,7 +83,13 @@ export class ResourceTemplateForm extends Component {
 
   resourceTemplatePromises = async (templateRefs) => {
     return Promise.all(templateRefs.map(rtId =>
-      getResourceTemplate(rtId)
+      getResourceTemplate(rtId).catch(err => {
+        const errId = decodeURIComponent(/[^/]*$/.exec(err.url)[0])
```

Comment:
while I laud the one-liner, I find this obtuse at best and would not say it passes the "inebriated coder" test.

Reply:
So given @mjgiarlo comments below I'm not sure if I should use regex in an imported "utilities" function. I ended up not doing that and used substring instead.